src/components/navigation-bar/Navigation-bar.js

```diff
@@ -25,23 +24,62 @@ const renderLeftDropdown = props => {
           data-testid="navbarDropdownSubmenuToggle"
         />
         <Dropdown.Menu data-testid="navbarDropdownSubmenu">
-          <Dropdown.Item>
-            <Link to="/canvas" className="navbar-dropdown-submenu">
-              Team Continuous
+          <LinkContainer to="/canvas" className="navbar-dropdown-submenu">
+            <Dropdown.Item>Team Continuous</Dropdown.Item>
+          </LinkContainer>
+          <LinkContainer to="/canvas-frontend" className="navbar-dropdown-submenu">
+            <Dropdown.Item>Team Frontend Auto</Dropdown.Item>
+          </LinkContainer>
+          <LinkContainer to="/canvas-mobile" className="navbar-dropdown-submenu">
+            <Dropdown.Item>Team Mobile</Dropdown.Item>
+          </LinkContainer>
+        </Dropdown.Menu>
+      </Dropdown>
+    )
+  }
+}
+
+const renderRightComponents = props => {
```

Comment:
renderRightComponents is now declared twice in this file, I'm guessing you just forgot to remove the other one (on line 87)

Reply:
I removed the wrong one